### ISSUE TITLE
Duplicate KeyUp events are raised for arrow keys

### ISSUE THREAD

**Issue by jeremy-visionaid** [microsoft/microsoft-ui-xaml#9399]
### Describe the bug

When an arrow key is pressed and released, the KeyUp event is raised twice

### Steps to reproduce the bug

```
        public MainWindow()
        {
            this.InitializeComponent();

            grid.KeyDown += Grid_KeyDown;

            grid.KeyUp += Grid_KeyUp;
        }

        private void Grid_KeyDown(object sender, KeyRoutedEventArgs e)
        {
            Debug.WriteLine($"KeyDown: {e.Key}");
        }

        private void Grid_KeyUp(object sender, KeyRoutedEventArgs e)
        {
            Debug.WriteLine($"KeyUp: {e.Key}");
        }
```

Press and release the "Up" arrow key. It then prints:

```
KeyDown: Up
KeyUp: Up
KeyUp: Up
```

### Expected behavior

The KeyUp event should only be raised once. i.e.

```
KeyDown: Up
KeyUp: Up
```

### NuGet package version

WinUI 3 - Windows App SDK 1.5.0: 1.5.240227000

### Windows version

Windows 11 (22H2): Build 22621

**Comment by jeremy-visionaid** [comment]
Repro available here:
https://github.com/molesmoke/WinUITestApp/tree/keyup

**Comment by codendone** [comment]
It looks like CXamlIslandRoot::PreTranslateMessage() results in delivering an CXamlIslandRoot::OnIslandKeyUp(), which gets handled as a normal key up, and then the standard key input still gets delivered to CXamlIslandRoot::OnIslandKeyUp(), causing it to be processed again. Something needs to change so only one or the other gets delivered/processed.

**Comment by jeremy-visionaid** [comment]
@codendone Maybe related if you're looking into this one, key down events are not raised for some combinations: #9399

**Comment by jeremy-visionaid** [comment]
Still present in v1.5.240607001

**Comment by codendone** [comment]
This is fixed in [1.6.240701003-experimental2](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.6.240701003-experimental2).
